AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -240,6 +258,14 @@ - (void)deleteNodesOfKind:(NSString *)kind atIndexPaths:(NSArray *)indexPaths co
   }];
 }
 
+- (void)deleteAllNodesOfKind:(NSString *)kind
+{
+  [_editingNodes[kind] removeAllObjects];
```

Comment:
No, I think. _externalCompletedNodes is only used during batch updates and should be managed entirely within endUpdatesAnimated:completion.